src/main/java/com/bearcat/bearcatwebpage/src/EventListingScreen.java

```diff
@@ -15,11 +17,20 @@ public EventListingScreen() {
         events.add("Event 3: Yet another event happening today.");
 
 
+
+
+
         JFrame frame = new JFrame("Bearcat Event Planner");
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setSize(400, 400);
 
 
+        Toolkit kit = Toolkit.getDefaultToolkit();
+        Dimension screenSize = kit.getScreenSize();
+        int screenHeight = screenSize.height;
+        int screenWidth = screenSize.width;
+        frame.setSize(3*(screenWidth / 4), 3*(screenHeight / 4));
+        frame.setLocationRelativeTo(null);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+

```

Comment:
Added the toolkit to not rely on the hardcoding of the screen size of the application in the future. This will further reduce technical debt in the project by reducing the amount of time the size needs to be adjusted. https://www.techopedia.com/definition/27913/technical-debt